youtube_transcript_api/formatters.py

```diff
@@ -0,0 +1,174 @@
+from collections import defaultdict
+import json
+import re
+
+
+def parse_timecode(time):
+    """Converts a `time` into a formatted transcript timecode.
+
+    :param time: a float representing time in seconds.
+    :type time: float
+    :return: a string formatted as a timecode, 'HH:MM:SS,MS'
+    :rtype str
+
+    :example:
+    >>> parse_timecode(6.93)
+    '00:00:06,930'
+    """
+    
+    time = float(time)
+    times = {
+        'hours': str(int(time) // 3600).rjust(2, '0'),
+        'mins': str(int(time) // 60).rjust(2, '0'),
+        'secs': str(int(time) % 60).rjust(2, '0'),
+        'ms': str(int(round((time - int(time))*1000, 2))).rjust(3, '0')
+    }
+    return "{hours}:{mins}:{secs},{ms}".format(**times)
```

Comment:
Why are you not using the `datetime` module?

Reply:
Good question! I didn't look into `datetime` yet but will definitely. I re-used an implementation seen in issue #23 from this person who displayed the start of what a formatter might look like:

https://github.com/jdepoix/youtube-transcript-api/issues/23#issuecomment-618953435

Reply:
Ah, okay. I am pretty sure that this can be done with `datetime`, which probably would be a bit more intuitive to read and less error prone, especially given how common `datetime` is in the python community.